Week1/assignments/usmaila-abdoul/Staircase.js

```diff
@@ -0,0 +1,16 @@
+const staireCase = (n) => {
+	const Arr = [];
+  
+  for (let i =0; i <= n; i++) {
+  	Arr.push(' ');
+  }
+  
+  for (let i = 0; i<= n; i++) {
+  	Arr.shift();
+    Arr.push('#');
+    var arr = Arr.join('')
+    console.log(arr)
```

Comment:
Your output isn't correct.
So for n = 6, your last line should have 6 #s and not space. 
for your solution, in the case where n = 6, the last line would have 7 #s